### ISSUE TITLE
fasta_to_features.py Biopython Deprecation Warning

### ISSUE THREAD

**Issue by ntseng57** [BinPro/CONCOCT#169]
Hi, 

When I use the code:
`$CONCOCT_dir/scripts/fasta_to_features.py 02_assembly_metaSPADES/spades_output_nocareful_k127/contigs.fasta 55215 4 05_binning_cocacola/kmer_4_tmp.csv`

I get this issue and no output:
```
BiopythonDeprecationWarning: This method is obsolete; please use str(my_seq) instead of my_seq.tostring().
  BiopythonDeprecationWarning)
```

Could you fix the fasta_to_features script? 

Thank you.

**Comment by jvollme** [comment]
While I also hope that concoct is going to be updated in the future, this warning just tells you that this code may not work in FUTURE Biopython versions. For now (as long as it only gives a DeprecationWARNING and not an ERROR) it should still work (similar issue was posted here: #133).
In case concoct is not going to be updated anytime soon, you can still maintain an old version of biopython for concoct without disturbing your other workflows, easy and comfortably using [bioconda](https://bioconda.github.io/)

**Comment by inodb** [comment]
Thanks @jvollme yes this indeed not an issue at the moment

**Comment by alneberg** [comment]
This was fixed in 966ed02e48df6dc17bd2b5632d32354010227d7c.
